packages/admin-next/dashboard/src/routes/orders/order-create-fulfillment/components/order-create-fulfillment-form/order-create-fulfillment-item.tsx

```diff
@@ -0,0 +1,149 @@
+import { useMemo } from "react"
+import { useAdminVariantsInventory } from "medusa-react"
+import { useTranslation } from "react-i18next"
+import { Trash } from "@medusajs/icons"
+import * as zod from "zod"
+
+import { LineItem } from "@medusajs/medusa"
+import { Input, Text } from "@medusajs/ui"
+import { UseFormReturn } from "react-hook-form"
+
+import { CreateFulfillmentSchema } from "./constants"
+import { Form } from "../../../../../components/common/form"
+import { getFulfillableQuantity } from "../../../../../lib/line-item"
+import { Thumbnail } from "../../../../../components/common/thumbnail"
+import { MoneyAmountCell } from "../../../../../components/table/table-cells/common/money-amount-cell"
+import { ActionMenu } from "../../../../../components/common/action-menu"
+
+type OrderEditItemProps = {
+  item: LineItem
+  currencyCode: string
+  locationId?: string
+  onItemRemove: (itemId: string) => void
+  form: UseFormReturn<zod.infer<typeof CreateFulfillmentSchema>>
+}
+
+export function OrderCreateFulfillmentItem({
+  item,
+  currencyCode,
+  form,
+  locationId,
+  onItemRemove,
+}: OrderEditItemProps) {
+  const { t } = useTranslation()
+
+  const { variant } = useAdminVariantsInventory(item.variant_id as string)
+
+  const hasInventoryItem = !!variant?.inventory.length
+
+  const { availableQuantity, inStockQuantity } = useMemo(() => {
+    if (!variant || !locationId) {
+      return {}
+    }
+
+    const { inventory } = variant
+
+    const locationInventory = inventory[0]?.location_levels?.find(
+      (inv) => inv.location_id === locationId
+    )
+
+    if (!locationInventory) {
+      return {}
+    }
+
+    return {
+      availableQuantity: locationInventory.available_quantity,
+      inStockQuantity: locationInventory.stocked_quantity,
+    }
+  }, [variant, locationId])
+
+  const minValue = 0
+  const maxValue = Math.min(
+    getFulfillableQuantity(item),
+    availableQuantity || Number.MAX_SAFE_INTEGER
+  )
+
+  return (
+    <div className="bg-ui-bg-subtle shadow-elevation-card-rest my-2 rounded-xl ">
+      <div className="flex gap-x-2 border-b p-3 text-sm">
+        <div className="flex flex-grow items-center gap-x-3">
+          <Thumbnail src={item.thumbnail} />
+          <div className="flex flex-col">
+            <div>
+              <Text className="txt-small" as="span" weight="plus">
+                {item.title}
+              </Text>
+              {item.variant.sku && <span>({item.variant.sku})</span>}
+            </div>
+            <Text as="div" className="text-ui-fg-subtle txt-small">
+              {item.variant.title}
+            </Text>
+          </div>
+        </div>
+
+        <div className="text-ui-fg-subtle txt-small mr-2 flex flex-shrink-0 flex-col items-center">
+          <MoneyAmountCell
+            className="justify-end"
+            currencyCode={currencyCode}
+            amount={item.total}
+          />
+          {hasInventoryItem && (
```

Comment:
comment: If the variant has an inventory item this section will be visible but if inventory doesn't have location levels we will display: `Available: N/A · In stock: N/A`. Do we want this or maybe to hide the section altogether in that case.